Classes/Controller/PostController.php

```diff
@@ -173,20 +175,34 @@ public function createAction(\AgoraTeam\Agora\Domain\Model\Post $newPost) {
 			);
 			$this->redirect('list', 'Post', 'agora', array('thread' => $newPost->getThread()));
 		}
-
 		$user = $this->getUser();
-
 		$newPost->setCreator($user);
 		$now = new \DateTime();
 		$newPost->setPublishingDate($now);
 		$this->postRepository->add($newPost);
-
 		$this->addFlashMessage(
 			\TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate('tx_agora_domain_model_post.flashMessages.created', 'agora'),
 			'',
 			\TYPO3\CMS\Core\Messaging\AbstractMessage::OK
 		);
-
+        if( ( $this->settings['post']['notificationsForPostOwner'] == 1 ) and  is_object($creator = $newPost->getQuotedPost()) )
+        {
+            $creator = $newPost->getQuotedPost()->getCreator();
+            $this->sendMail(
+                array(
+                    $creator->getEmail() => $creator->getDisplayName()
+                ),
+                array(
+                    $this->settings['post']['defaultPostEmailAdress'] =>  $this->settings['post']['defaultPostEmailUserName']
```

Comment:
maybe we can implement a controller method like $this->getDefaultSender() that returns an array like it is needed here? only for better encapsulation